.gitignore

```diff
@@ -1 +0,0 @@
-*.txt
```

Comment:
How come you deleted the `.gitignore`? It looks you need to add `.DS_Store` to it so that those files aren't tracked by git.